doc/api/util.md

```diff
@@ -207,6 +207,23 @@ Values may supply their own custom `inspect(depth, opts)` functions, when
 called these receive the current `depth` in the recursive inspection, as well as
 the options object passed to `util.inspect()`.
 
+### util.inspect.config([options])
```

Comment:
Hmm, I think it might be possible to make them plain properties. Would save some time having to compute the getter/setter names at least.